src/index.html

```diff
@@ -11,12 +11,100 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/home"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo-image"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              href="/apple"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav-item">
```